pkgs/development/libraries/qt-5/5.14/srcs.nix

```diff
@@ -6,8 +6,8 @@
   qt3d = {
     version = "5.14.2";
     src = fetchurl {
-      url = "${mirror}/official_releases/qt/5.14/5.14.2/submodules/qt3d-everywhere-src-5.14.2.tar.xz";
-      sha256 = "9da82f1cc4b7d416d31ec96224c59d221473a48f6e579eef978f7d2e3932c674";
+      url = "${mirror}/archive/qt/5.14/5.14.2/submodules/qt3d-everywhere-src-5.14.2.tar.xz";
+      sha256 = "0x6668wjwzcgjzprwmvfiyj76512kp2j8qn93v9idm5pqhf2za4x";
```

Comment:
Why does the hash change?

Reply:
I initially thought sha256 depends on URL. But it does not. Looking at hash length it's just a matter of the default:

```
$ nix hash to-base16 sha256:0x6668wjwzcgjzprwmvfiyj76512kp2j8qn93v9idm5pqhf2za4x
9da82f1cc4b7d416d31ec96224c59d221473a48f6e579eef978f7d2e3932c674
```

Given that hash won't change I'll regenerate all URLs in that file (and keep initial hashes).

Reply:
Regenerated all URLs with base16 hashes. None changed \o/.